### DISCUSSION TITLE
Remote Control Plugin - Objects Info

### DISCUSSION THREAD

**Discussion by Xio1996** [Stellarium/stellarium#3604] [Q&A]
Hi Everyone,

I am using the Remote Control Plugin (/api/objects/info?format=map) to return data on the selected object in Stellarium. The object is then added to an AstroPlanner plan. I'm having trouble getting the objects size/orientation information returned.

As an example I selected NGC 6740 (galaxy) and requested the data in JSON format. The size information for the major axis is returned in several formats.

    "size": 0.0003470296231767495,
    "size-dd": 0.019883332774043083,
    "size-deg": "0.01988°",
    "size-dms": "+0°01'11.58\"",

However, Stellarium displays not only the major axis but also the minor axis and orientation angle. I know that if I call the 'info' command on the Remote Control Plugin, without a format parameter, then I will get back the displayed html code. 

Size: +0Â°01'11.58" x +0Â°00'41.52"<br />Orientation angle: 40Â°

Is it possible to get this information (if available) in the JSON/map format? 

Many thanks for your help.

Pete

**Comment by gzotti** [comment]
Sure, it's just not implemented.

**Comment by gzotti** [comment]
Continued in #3605
